notebooks/official/feature_store/online_feature_serving_and_fetching_bigquery_data_with_feature_store.ipynb

```diff
@@ -95,11 +95,11 @@
       },
       "source": [
         "### Known Issues\n",
-        "This public Preview release has the following known issues:\n",
+        "This service has the following known issues:\n",
```

Comment:
The whole known issue should be removed.

Reply:
Removed from all notebooks